community-membership.md

```diff
@@ -113,26 +113,27 @@ only be doing issues triage and have no approval rights.
 ### Requirements
 
 The following apply to the part of the codebase for which one would be an
-approver in the `CODEOWNER` files.
+approver in the `CODEOWNERS` files.
 
-- Reviewer of the codebase for at least 3 months
-- Reviewer for at least 10 substantial PRs to the codebase
-- Reviewed or merged at least 30 PRs to the codebase
+- Reviewer of the codebase for at least 1 month
+- Reviewer for or author of at least 10 substantial PRs to the codebase,
+  with the definition of substantial subject to the maintainer's discretion
+  (e.g. refactors/adds new functionality rather than one-line pulls).
 - Nominated by a maintainer
   - With no objections from other maintainers
-  - Done through PR to update the `CODEOWNER`.
+  - Done through PR to update the `CODEOWNERS`.
 
 ### Responsibilities and privileges
 
 The following apply to the part of the codebase for which one would be an
-approver in the `CODEOWNER` files.
+approver in the `CODEOWNERS` files.
 
 - Approver status may be a precondition to accepting large code contributions
```

Comment:
not sure what this means

Reply:
this is a minor diff to the existing text, just properly using `CODEOWNERS` instead of `CODEOWNER`.